src/schemas/22-0993/schema.js

```diff
@@ -0,0 +1,37 @@
+import _ from 'lodash';
+import definitions from '../../common/definitions';
+import schemaHelpers from '../../common/schema-helpers';
+
+let schema = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'REQUEST TO OPT-OUT OF INFORMATION SHARING WITH EDUCATIONAL INSTITUTIONS (VA FORM 22-0993)',
+  type: 'object',
+  definitions: _.pick(definitions, [
+    'fullName',
+    'ssn',
+    'vaFileNumber',
+    'privacyAgreementAccepted'
+  ]),
+  additionalProperties: false,
+  properties: {},
+  oneOf: [
```

Comment:
Is 'oneOf' good, or should I use 'anyOf'?

Reply:
are people allowed to enter both?

Reply:
good question.  The doc I linked to says:

* Social security number [number input] [required]
* I don't have an SSN [checkbox] [optional]
  * VA file number [revealed and required if I don't have an SSN is checked]

So, theoretically they would only enter one, but I can't think of a scenario where it would hurt if they entered both for some reason.  I'll change it to 'anyOf'